Makefile

```diff
@@ -0,0 +1,10 @@
+#/bin/bash
+
```

Comment:
> I normally directly use the make file commands in my travis file if possible

OK I understand now.
Keep the makefile.

You can also add a `.travis.yml` file base on my other projects:
https://github.com/pierrre/githubhook/blob/master/.travis.yml